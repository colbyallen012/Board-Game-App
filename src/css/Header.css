h1 {
  color: #444f68;
  margin: 0;
  font-size: 50px;
  text-align: center;
  letter-spacing: 3px;
  text-shadow: 3px 3px #2eaebb;
}

.title-style {
  font-size: 100px;
}

header {
  background-image: url(https://images.pexels.com/photos/776654/pexels-photo-776654.jpeg?cs=srgb&dl=blur-board-game-cards-776654.jpg&fm=jpg);
  background-size: cover;
  height: 250px;
  width: 100%;
  background-repeat: no-repeat;
  background-position: 50% 68%;
}
