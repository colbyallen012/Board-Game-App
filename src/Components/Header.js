import React from 'react';
import '../css/Header.css'

function Header() {
  return (
    <header>
      <h1><span className="title-style">Board,</span><br/> not bored</h1>
    </header>
  )
}

export default Header;
