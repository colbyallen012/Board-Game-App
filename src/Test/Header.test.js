import React from 'react';
import ReactDOM from 'react-dom';
import Header from '../Components/Header';


describe('Header', () => {
  let wrapper;
  
  beforeEach(() =>{
    wrapper = shallow(
      <Header/>
    )
  })

  it('should match the snapshot with all data passed in correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});