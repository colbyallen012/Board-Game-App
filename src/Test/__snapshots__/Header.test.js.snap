// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match the snapshot with all data passed in correctly 1`] = `
<header>
  <h1>
    <span
      className="title-style"
    >
      Board,
    </span>
    <br />
     not bored.
  </h1>
</header>
`;
